internal/resource/url.go

```diff
@@ -219,9 +219,13 @@ func (f *Fetcher) FetchFromTFTP(u url.URL, dest *os.File, opts FetchOptions) err
 // FetchFromHTTP fetches a resource from u via HTTP(S) into dest, returning an
 // error if one is encountered.
 func (f *Fetcher) FetchFromHTTP(u url.URL, dest *os.File, opts FetchOptions) error {
+
 	if f.client == nil {
+		logger := log.New(true)
```

Comment:
why this addition?

Reply:
f enters FetchfromHTTP as nil, for Looger and client

Reply:
Rather than instantiating the logger here would it work to change [the Fetcher instantiation here](https://github.com/Bubblemelon/ignition/blob/f59dbcdbcac1eda691d1ef52f7ce2b537a1a1c00/internal/oem/oem.go#L68) to call `rf, err := c.NewFetcherFunc(log.New(true))`?